src/deluge/gui/views/performance_session_view.cpp

```diff
@@ -605,13 +605,13 @@ void PerformanceSessionView::renderFXDisplay(params::Kind paramKind, int32_t par
 					buffer = "8ths";
 				}
 				else if (knobPos < 14) { // 16ths stutter: 2 leds turned on
-					buffer = "16ths";
+					buffer = "16th";
```

Comment:
These labels are all incorrect at non default resolutions - this bug was fixed in the main quantized stutter before Amadeus so you should be able to reuse that logic